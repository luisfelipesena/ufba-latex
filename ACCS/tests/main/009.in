7800 16695590 10783579
