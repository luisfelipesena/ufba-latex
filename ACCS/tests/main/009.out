180038213724410
